testcases/replicaset/replicaset_kw.robot

```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Library           Collections
+Library           RequestsLibrary
+# For regular execution
+#Library           KubeLibrary
```

Comment:
Please change the library import to 
```
Library    KubeLibrary
```
This way, this example will work once the keyword is added to the released version.